runlog.c

```diff
@@ -312,6 +312,7 @@ run_add_record(
         int            eoln_type,
         int            variant,
         int            is_hidden,
+        int            upsolving,
```

Comment:
I think that check in 'run_add_record' should be removed. The submission time check is performed before calling run_add_record in all cases anyway.